src/Components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import SingleBlock from './SingleBlock/SingleBlock';
+import data from './SingleBlock/data';
+import star from '@/images/Homepage/star.svg';
+import './style.scss';
+
+interface DataItem {
+  id: number;
+  number: string;
+  title: string;
+  text: string;
+}
+
+const Accordion: React.FC = () => {
+  const [accordionItem, setAccordionItem] = useState<DataItem[]>(data);
```

Comment:
you are not using `setAccordionItem`; 

p.s. your vscode does not show the lint errors  like this ?
<img width="587" alt="image" src="https://github.com/user-attachments/assets/77767395-6d92-4d35-b3fb-f74cabed9f58">

Reply:
I thought that only one part of useState could be used.  There was a red line, but the code worked. Lesson learned. Since the state does not change, useState was removed. changed to  - const accordionItem: DataItem[] = data;